src/pages/error-page/error-page.js

```diff
@@ -4,15 +4,14 @@ import Button from '@material-ui/core/Button';
 import { useStyles } from './error-page.styles';
 import { config } from '../../configs';
 
-const { ERROR_PAGE_STATUS } = config.statuses;
+const { ERROR_BOUNDARY_STATUS } = config.statuses;
 
 const ErrorPage = () => {
   const classes = useStyles();
 
   return (
     <div className={classes.container}>
-      <h1>404</h1>
-      <h3>{ERROR_PAGE_STATUS}</h3>
+      <h2>{ERROR_BOUNDARY_STATUS}</h2>
```

Comment:
We have 2 errors `ERROR_PAGE_STATUS `and `ERROR_BOUNDARY_STATUS`. So why you remove `ERROR_PAGE_STATUS`?

Reply:
As we discuss at Friday meeting it should display another text when an error comes up

Reply:
@moran711 or you want to show different errors depends on which error type is coming up?

Reply:
It should display another error message on error page for error boundary, but not every time.

Reply:
Got it, I'll make ternary operator than

Reply:
done, thanks for comment